lib/array.js

```diff
@@ -64,10 +64,26 @@ const sequence = (seq, max) => {
 // Returns: <any>, element
 const last = arr => arr[arr.length - 1];
 
+// Push single value multiple times
+//   arr - <Array>
+//   n - <number>
+//   value - <any>
+// Returns: <number>, new value of arr.length
+const pushSame = (arr, n, value) => {
```

Comment:
`pushMultiple`?

Reply:
@nechaido the function pushes the same value 0 or more times, thus I think naming it `pushMultiple` would be ambiguous.

Reply:
@belochub I do not like `pushSame`, but I do not have any better option, so you can proceed and land it.